tests/integration_test.rs

```diff
@@ -0,0 +1,42 @@
+#![warn(clippy::pedantic)]
+
+use libcnb_test::{BuildpackReference, IntegrationTest};
+use std::io;
+
+#[test]
+#[ignore]
+fn test() {
+    IntegrationTest::new("heroku/buildpacks:20", "tests/fixtures/default_ruby")
+        .buildpacks(vec![BuildpackReference::Crate])
```

Comment:
This `.buildpacks()` call is equivalent to the default value, so isn't needed:
https://github.com/Malax/libcnb.rs/blob/5b9495d84f605018f9ff61c3b72ca422d975acde/libcnb-test/src/lib.rs#L118

```suggestion
```